index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
```

Comment:
Great job describing the changes well but please consider always clarifying the reasons for a change means describing clearly to the user why the change is necessary or being asked.

## Example:- 

> Please consider making the font-awesome link for icons works properly to make it professional and to give a good user experience.